김민진/Bank/src/Bank/Main.java

```diff
@@ -0,0 +1,19 @@
+package Bank;
+
+public class Main {
+	
+	public Main() {
+		
+	}
+	
+	public void run() {
+		BankApplication bankApplication = new BankApplication();
+	}
+	
+	public static void main(String[] args) {
+		Main main = new Main();
+		main.run();
```

Comment:
```suggestion
		BankApplication bankApplication = new BankApplication();
```
main 클래스 안에서 해결해도 될 것 같습니다.
분리할 필요가 없을 것 같습니다